### DISCUSSION TITLE
Arista NG Firewall Syslog Ingestion

### DISCUSSION THREAD

**Discussion by Matt-CyberGuy** [Security-Onion-Solutions/securityonion#12409] [Unsupported Versions]
Hi Everyone,
The MSSP I work for has been working on building out/utilizing security onion to build up our security monitoring toolset. We manage and deploy networks with palo alto, and a handful of different platforms. We use Arista NG Firewalls for a number of our smaller clients. I saw Elastic has an integration for this, but didn't see it in the deployment i set up last night. Is there a way to include this?

https://docs.elastic.co/integrations/arista_ngfw

-Matt

**Comment by reyesj2** [comment]
We will definitely take a look at additional integrations as time allows. Preference is given to integration requests from our paid customers.

**Comment by Matt-CyberGuy** [comment]
All good, we're interested in becoming a paying customer, but there is a definitely a steep learning curve to this platform. I'm trying to figure out the best way to test/build this out to show it's value to include it in our service offerings.
